views/membership.py

```diff
@@ -6,6 +8,8 @@
 from groups.permissions.admin import HasAdminRights, has_admin_rights
 from groups.serializers.membership import MembershipSerializer
 
+logger = logging.getLogger('groups')
```

Comment:
Leave two lines after imports.

Reply:
@shreyansh23 I can't apologize enough for this. This was a wrong suggestion. There is no rule that says leave two lines after imports.

Reply:
No problem! :stuck_out_tongue: